server/engine/src/main/java/org/eclipse/lsp/cobol/lsp/handlers/text/DidCloseHandler.java

```diff
@@ -56,6 +61,13 @@ public void didClose(DidCloseTextDocumentParams params) throws InterruptedExcept
     LOG.info(format("Document closing invoked on URI %s", uri));
     watcherService.removeRuntimeWatchers(uri);
     documentModelService.closeDocument(uri);
+    if (copybookService instanceof CopybookServiceImpl) {
```

Comment:
wanted to add below method to the CopybookService, but that would mean update in common module
```java
  /**
   * Removes cache for the passed {@link CopybookId}
   * @param copybookId
   */
  void invalidateCache(CopybookId copybookId) ;

  /**
   * Get the list of copybook used by a document
   *
   * @param documentUri  current document uri.
   * @return List of all the {@link CopybookModel} used by the passed document
   */
  List<CopybookModel> getCopybookUsage(String documentUri);
```